.github/workflows/merge.yml

```diff
@@ -0,0 +1,24 @@
+name: merge
+
+# open or update publishing PR when there is a push to main
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
My guess is it will incrementally update the existing PR if already opened right?